docs/topics/recipes.md

```diff
@@ -743,7 +743,7 @@ easy manner.
 ### Valid array keys for style `applyFromArray()`
 
 The following table lists the valid array keys for
-\PhpOffice\PhpSpreadsheet\Style applyFromArray() classes. If the "Maps
+Style applyFromArray() classes. If the "Maps
```

Comment:
I would tend to agree that fully qualified class names might be better for documentation. That way the user don't need to look for where the class lives.

I guess we could have exceptions to that in specific context where it is rather obvious, but most class names in documentation should probably stay in FQCN form.